src/main/java/ui/components/MainPanel.java

```diff
@@ -4,23 +4,15 @@
 import java.awt.*;
 
 public class MainPanel extends JPanel {
```

Comment:
While I don't think that a Doc would do much for this function I would still add one. Better play it safe